storage/integration-test/src/storage.test.js

```diff
@@ -28,18 +28,18 @@ describe('Storage', () => {
   }, 90000)
 
   afterAll(async () => {
-    if(amqpConnection) {
-      console.log("Closing AMQP connection...")
+    if (amqpConnection) {
+      console.log('Closing AMQP connection...')
```

Comment:
How are these `console.log()` provide a useful information. It is a tear-down function and if something goes wrong here the test will end anyways. I would remove the `console.log()` statements to reduce clutter.